prowler/providers/aws/services/ssmincidents/ssmincidents_enabled_with_plans/ssmincidents_enabled_with_plans.py

```diff
@@ -0,0 +1,27 @@
+from prowler.lib.check.models import Check, Check_Report_AWS
+from prowler.providers.aws.services.ssmincidents.ssmincidents_client import (
+    ssmincidents_client,
+)
+
+
+class ssmincidents_enabled_with_plans(Check):
```

Comment:
What is the relation between`ssmincidents_enabled_with_plans` enabled and the SSM replication sets? Just curious.

Reply:
Cool, understood!